tests/main.test.ts

```diff
@@ -44,7 +46,20 @@ describe("User start/stop", () => {
 
     const { content } = await userStartStop(context);
 
-    expect(content).toEqual("Task assigned successfully");
+    expect(content).toEqual(SUCCESS_MESSAGE);
+  });
+
+  test("User can start an issue with trimmed command", async () => {
+    const issue = db.issue.findFirst({ where: { id: { equals: 1 } } }) as unknown as Issue;
+    const sender = db.users.findFirst({ where: { id: { equals: 1 } } }) as unknown as PayloadSender;
```

Comment:
This uses a database? It should be querying the GitHub api only.

Reply:
This is a db used for faking API calls: https://github.com/ubiquibot/command-start-stop/blob/19743ea67faaaea32937791787f000701b8d1cb5/tests/main.test.ts#L3

Reply:
That doesn't seem to make sense to me. We should be using a mock http server not a database.

Reply:
No idea about all this